python/scenario/scenario_executor.py

```
@@ -1898,6 +1895,76 @@ def _emit_message_snapshot_event(self, scenario_run_id: str) -> None:
+        the stream-completion side effects.
+        """
+        if self._finished_emitted:
+            return
+        self._emit_run_finished_event(scenario_run_id, result, status)
+
+    def _minimal_error_result(self, error: BaseException) -> ScenarioResult:
+        """Smallest valid ERROR result, for when building the full one fails."""
+        return ScenarioResult(
+            success=False,
+            messages=[],
+            reasoning=f"Scenario failed with error: {str(error)}",
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)